recipe/slack.php

```diff
@@ -31,10 +31,13 @@
     }
 
     $attachment = [
+        'username' => get('slack_username'),
         'title' => get('slack_title'),
+        'pretext' => get('slack_pretext'),
```

Comment:
What if user didn't specified slack_pretext? Same for slack_icon.

Reply:
My apologies, I overlooked this. According to the slack api documentation the value of each property can be an empty string (" "). But I gave the icon and username a default value.